service/src/test/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiControllerGcsBucketTest.java

```diff
@@ -142,8 +143,10 @@ public void setup() throws Exception {
                 STORAGE_CLASS,
                 LIFECYCLE_API)
             .getGcpBucket();
-    cloudUtils.addFileToBucket(
-        userAccessUtils.defaultUser().getGoogleCredentials(), projectId, sourceBucketName);
+    GcpCloudUtils.runWithRetryOnException(
+        () ->
+            cloudUtils.addFileToBucket(
```

Comment:
can we add the retry inside addFileToBucket here https://github.com/DataBiosphere/terra-workspace-manager/blob/3bd1f03cd912ea8fe93e5678a7b97876df71f1a4/service/src/test/java/bio/terra/workspace/common/GcpCloudUtils.java#L139

Reply:
Good call, done.